platform_view_swift/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: platform_view_swift
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  collection: 1.14.6 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
These version numbers (and the comments next to them) are holdovers from running `flutter update-packages`, which isn't the recommended approach for typical apps, so they should probably just be re-pinned manually. 

Actually, now that I'm looking at these deps, are any of these being used anywhere in this sample? It's possible they were copied over from another sample when the the original, Obj-C platform view sample was made, and they're no longer necessary.

Try cutting them all out and see if you can still build. :)

Reply:
You were right -- not needed.